### ISSUE TITLE
Project/index settings show insuficient hints on metadata

### ISSUE THREAD

**Issue by McSinyx** [pypa/warehouse#7892]
First of all, sorry for not following any template, since I don't know where would this fit in.

![2020-05-05T20:23:53](https://user-images.githubusercontent.com/13689192/81071344-c8d5f280-8f0e-11ea-9fd1-cd89fae09316.png)

I am referring to the

> To set the '...' description, author, links, classifiers, and other details for your next release, use the `setup()` arguments in your `setup.py` file.

With multiple build backends are now available, I think this section should include them too.  For the matter of completeness, the Python Packaging User Guide does link to flit, hatch and poetry, so I'm not sure if not mentioning other backends is by design though.

In addition, `setuptools` has been supporting `setup.cfg` for quite a while now, and I wonder if we want to rephrase this entire paragraph more generically.

**Comment by di** [comment]
I agree, but I think this information is probably a little too specific: instead of including more options here, we should include less. For example, the "*must* use Twine" part is not really true, I'd say that we'd recommend users use Twine, but it's not a requirement.

This infobox is there because users were looking for the ability to "edit" the details of their release on PyPI, and we need to direct them to "edit your release by changing it and making a new release". I think we can probably get that point across more succinctly without exploding this into a description of N different tools.

**Comment by vivekvj18** [comment]
@di 
@McSinyx 
Can I Take This Task...?

**Comment by McSinyx** [comment]
Yes, please go ahead and grab this issue! At least within Warehouse, our rule is: If there is not a corresponding pull request for this issue (visibly linked by issue number/PR number), and it's not marked as assigned to anyone, it is up for grabs. So in cases like this, you don't even need to ask. :) Just comment and say that you're starting to work on it, and link in a comment to your work-in-progress branch once you start.

As always, if you have questions along the way as you work on this, please feel free to ask them here, in [`#pypa-dev` on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or the [pypa-dev mailing list](https://groups.google.com/forum/#!forum/pypa-dev). Thank you!

_Originally posted by @brainwane in https://github.com/pypa/warehouse/issues/6552#issuecomment-539179175_

In my own words, this means if I could rephrase that piece of doc properly then I would have made a PR earlier :smile:  Thanks for offering your help here, otherwise this low-priority issue would have to stall for quite some time!

**Comment by vivekvj18** [comment]
@McSinyx 
So I Can Take This Task Or Not...?

**Comment by McSinyx** [comment]
:smile: sorry if my phrasing was unclear, yes please!

**Comment by vivekvj18** [comment]
@McSinyx 
From What Is Name Of The File Which Is To Be Changed....?

**Comment by McSinyx** [comment]
[A quick search](https://github.com/pypa/warehouse/search?q=%22and+other+details+for+your+next+release%22&type=Code) shows the message is located at `warehouse/templates/manage/settings.html`.  Please don't forget to update the test as well.

**Comment by vivekvj18** [comment]
In Above PR I had Removed Must

**Comment by vivekvj18** [comment]
@McSinyx 
In Above PR I had removed must

**Comment by vivekvj18** [comment]
Actually i had attached a PR.
Kindly check and revert back if it needs any changes

**Comment by deveshks** [comment]
Hi @vivekvj18 

I see that you have opened the PR against your fork. I think you need to open the PR against this repo instead.  [Creating a pull request from a fork](https://help.github.com/en/enterprise/2.16/user/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) should help you on how to do that.

You can also take a look at [Warehouse:Getting Started](https://warehouse.pypa.io/development/getting-started/) for general directions on how to set up the development environment locally.

**Comment by vivekvj18** [comment]
Thank You @deveshks 
@McSinyx In the above Pr I had removed must

**Comment by McSinyx** [comment]
@vivekvj18, I don't think just removing *must* sufficiently address what @di meant by

> instead of including more options here, we should include less

The part of

> use the `setup()` arguments in your `setup.py` file

also needs some phrasing to be more generic too.
